src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -182,7 +189,7 @@ def _search(
                 query_kwargs.pop("sort_by")
                 result = inbox_search(**query_kwargs)
             else:
-                query_kwargs["referrer"] = "search.group_index"
+                query_kwargs["referrer"] = referrer
```

Comment:
Should we default to `DEFAULT_REFERRER` here too?

Reply:
referrer will always be passed in and defined here -- the default is set in the callers